lib/peeps.rb

```diff
@@ -0,0 +1,40 @@
+require 'pg'
+
+class Peeps
+  attr_reader :id, :username, :peep, :lastmodified
+
+  def initialize(id: , username: , peep: , lastmodified: ) 
+    @id = id
+    @username = username
+    @peep = peep
+    @lastmodified = lastmodified
+  end
+
+  def self.all 
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    result = connection.exec("SELECT * FROM peeps")
+    result.map do |peep|
+      Peeps.new(id: peep['id'], username: peep['username'],peep: peep['peep'], lastmodified: Time.parse(peep['lastmodified']).strftime("%d/%m/%Y %k:%M"))
+    end
+  end
+
+  def self.create(username: , peep: )
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    connection.exec("INSERT INTO peeps (username, peep) VALUES('#{username}', '#{peep}')")
+  end
+
+  # def time_formatter
```

Comment:
Really clean looking and functional code  in general - maybe you can space out the Peeps.new to be on multiple lines to really perfect this though